core/state/statedb.go

```diff
@@ -87,6 +87,18 @@ func (self *StateDB) GetLogs(hash common.Hash) vm.Logs {
 	return self.logs[hash]
 }
 
+func (self *StateDB) GetAllLogs() *map[common.Hash]vm.Logs {
```

Comment:
I think these were added in Gus' commit. I'll remove them in the second commit. This method isn't needed (anymore)